src/database/data-source.ts

```diff
@@ -0,0 +1,19 @@
+import "dotenv/config"
+import "reflect-metadata";
+import { DataSource } from "typeorm";
+import config from "@/config";
+
+export const AppDataSource = new DataSource({
+    ...config.db,
+    synchronize: false,
+    logging: config.env.isProd,
```

Comment:
this sef should be isdev, you dont do sensitive loggings like this in prod else hackers might get a structure of your system/database. we'll talk proper logging when the time comes